scripts/verify

```diff
@@ -0,0 +1,177 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+for i in "$@"
+do
+case $i in
+  --deployer=*)
+    deployer="${i#*=}"
+    shift
+    ;;
+  --parameters=*)
+    parameters="${i#*=}"
+    shift
+    ;;
+  --wait_timeout=*)
+    wait_timeout="${i#*=}"
+    shift
+    ;;
+  *)
+    echo "Unrecognized flag: $i"
+    exit 1
+    ;;
+esac
+done
+
+[[ -z "$deployer" ]] && deployer="$APP_DEPLOYER_IMAGE"
+[[ -z "$parameters" ]] && parameters="{}"
+[[ -z "$wait_timeout" ]] && wait_timeout=600
+
+# Set up kubectl configuration.
+setup_kubeconfig.sh
+
+# Extract schema and values files.
+docker run \
+    --entrypoint=/bin/bash \
+    --rm "${deployer}" \
+    -c 'cat /data/schema.yaml' \
+> /data/schema.yaml
+echo "$parameters" \
+  | json2yaml \
+> /data/values.yaml
+
+# Compose test id.
+test_id="$(cat /dev/urandom \
+    | tr -dc 'a-z0-9' \
+    | head -c 8 \
+|| true)"
+
+# Extract keys for name and namespace.
+name_key="$(extract_schema_key.py \
+    --schema_file=/data/schema.yaml \
+    --type NAME)"
+namespace_key="$(extract_schema_key.py \
+    --schema_file=/data/schema.yaml \
+    --type NAMESPACE)"
+
+export NAME="apptest-$test_id"
+export NAMESPACE="apptest-$test_id"
+
+# Stitch in name and namespace parameters.
+parameters=$(echo "$parameters" \
+  | jq \
+    --arg name_key "$name_key" \
+    --arg name "$NAME" \
+    --arg namespace_key "$namespace_key" \
+    --arg namespace "$NAMESPACE" \
+    '.[$name_key] = $name | .[$namespace_key] = $namespace')
+
+echo "INFO Parameters: $parameters"
+
+echo "INFO Creates namespace \"$NAMESPACE\""
+kubectl create namespace "$NAMESPACE"
+
+function delete_namespace() {
+  echo "INFO Collecting events for namespace \"$NAMESPACE\""
+  kubectl get events --namespace="$NAMESPACE" || echo "ERROR Failed to get events for namespace \"$NAMESPACE\""
+
+  if [[ ! -z $deployer_name ]]; then
+    echo "INFO Collecting logs for deployer"
+    kubectl logs "jobs/$deployer_name" --namespace="$NAMESPACE" || echo "ERROR Failed to get logs for deployer $deployer_name"
+  fi
+
+  echo "INFO Deleting namespace \"$NAMESPACE\""
+  kubectl get all --namespace "$NAMESPACE" --output=yaml
+  kubectl delete namespace "$NAMESPACE"
+}
+
+function clean_and_exit() {
+  message=$1
+  [[ -z "$message" ]] || echo "$message"
+
+  delete_namespace
+  exit 1
+}
+
+echo "INFO Initializes the deployer container which will deploy all the application components"
+/scripts/start_internal.sh \
```

Comment:
Can we update any remaining references in `verify` and `install` so that we can get rid of `driver_internal.sh` and `start_internal.sh` altogether? These are not part of the deployer APIs so I don't see a reason to keep them around.

Reply:
I thought I had, but missed this. Good catch!